uaclient/cli.py

```diff
@@ -1581,6 +1638,27 @@ def action_status(args, *, cfg):
     return ret
 
 
+def action_system(args, *, cfg, **kwargs):
+    """Perform the system action.
+
+    :return: 0 on success, 1 otherwise
+    """
+    _print_help_for_subcommand(
+        cfg, cmd_name="system", subcmd_name=args.command
+    )
+    return 0
+
+
+def action_system_reboot_required(args, *, cfg: config.UAConfig):
+    result = call_api("u.pro.security.status.reboot_required.v1", [], cfg=cfg)
+    event.info(
+        result.to_dict()
+        .get("data", {})
+        .get("attributes", {})["reboot_required"]
+    )
```

Comment:
if we call the function directly, we can use the typed python object instead of converting to a dict
```suggestion
    event.info(result.reboot_required)
```